cypress/integration/login/signup_spec.js

```diff
@@ -13,6 +13,13 @@ let config;
 
 describe('Signup Email page', () => {
     before(() => {
+        // Disable other auth options
+        const newSettings = {
+            Office365Settings: {Enable: false},
+            LdapSettings: {Enable: false},
```

Comment:
This is disabling the auth options we're using in the test server.  I'm thinking to disable that on the test server config so that we'll not worry in doing this during local dev. But I'll leave it as is for the moment.